src/bots/depositor_bot.py

```diff
@@ -70,18 +70,26 @@ def __init__(self, w3: Web3):
         self.deposit_prefix = contracts.deposit_security_module.functions.ATTEST_MESSAGE_PREFIX().call()
         logger.info({'msg': 'Call `ATTEST_MESSAGE_PREFIX()`.', 'value': str(self.deposit_prefix)})
 
+        transports = []
+
+        if variables.MESSAGE_TRANSPORT in [TransportType.RABBIT, TransportType.BOTH]:
+            transports.append(RabbitProvider(
+                client='depositor',
+                routing_keys=[MessageType.PING, MessageType.DEPOSIT],
+                message_schema=Schema(Or(DepositMessageSchema, PingMessageSchema)),
+            ))
+
+        if variables.MESSAGE_TRANSPORT in [TransportType.KAFKA, TransportType.BOTH]:
+            transports.append(KafkaMessageProvider(
+                client=f'{variables.KAFKA_GROUP_PREFIX}deposit',
+                message_schema=Schema(Or(DepositMessageSchema, PingMessageSchema)),
+            ))
+
+        if len(transports) < 1:
+            logger.warning({'msg': 'No transports found.'})
```

Comment:
Maybe raise exception here?
Not sure when we want to run depositor bot without any transport